### ISSUE TITLE
Increase pattern rigor?

### ISSUE THREAD

**Issue by mirh** [jonathanlermitage/tikione-steam-cleaner#26]
Removing whole folders only after a simple name check seems a bit too hazardous. 

For example: some game may even legitly have some folder named "directx" (for reasons). 
A check for DXSetup inside these would really nullify chances of messing up. 

And I guess the same is true for Physx and PhysX_systemsoftware. 

I guess one could then point out "vcredit" is all but a "random" name.. Though I'd still find the "file whitelist" approach more elegant.

**Comment by jonathanlermitage** [comment]
Patterns checks files/folders absolute files ; you can improve the whitelist : https://github.com/jonathanlermitage/tikione-steam-cleaner/blob/master/dist2/conf/tikione-steam-cleaner_dangerous-items.ini

**Comment by mirh** [comment]
I honestly don't understand what should be the point of that list. 
Why should the program check in windows folder (or anything other than [steamapps](27)) in the first place?

**Comment by jonathanlermitage** [comment]
Some users add "C:\" as a custom folder ;-)

**Comment by mirh** [comment]
> Some users add "C:\" as a custom folder ;-)

Oh... *Right*.. 
Never overestimate the user. 
Still, I don't see problems arising in a million of years, should you implement "strict" folder check. 
Aka "don't delete if there aren't [strictly only?] the files you'd expect that folder to contain". 

> then removed because it's not stable

How?

**Comment by jonathanlermitage** [comment]
patterns are now case-insensitive ; closing issue

**Comment by mirh** [comment]
> patterns are now case-insensitive ; closing issue

That was another issue tbh. 
Here I was requesting a stricter rigor for deletion. 
Like checking for specific files inside matching folders, or now that I think, checking is files have any special sign or hash.
